about.html

```diff
@@ -119,6 +120,7 @@ <h2 class="previous-course_title">Our academic history</h2>
                 </li>
             </ul>
         </section>
+        
         <footer>
             <img class="footer_img" src="images/codersblack.png" alt="coders circle logo">
             <p class="footer_p">2022 Coders Circle. All Rights Reserved</p>
```

Comment:
[REQUIRED] The footer for your about page should have a white background in mobile screen sizes and dark background in desktop screen sizes